onlineweb4/settings/example-local.py

```diff
@@ -146,17 +146,17 @@
 # Online stripe keys.
 # For development replace with https://online.ntnu.no/wiki/komiteer/dotkom/aktuelt/onlineweb4/keys/
 # For production login to Stripe
-STRIPE_PUBLIC_KEYS = [
-    "pk_test_replace_this",  # arrKom
-    "pk_test_replace_this",  # proKom
-    "pk_test_replace_this",  # triKom
-]
-
-STRIPE_PRIVATE_KEYS = [
-    "sk_test_replace_this",  # arrKom
-    "sk_test_replace_this",  # proKom
-    "sk_test_replace_this",  # triKom
-]
+STRIPE_PUBLIC_KEYS = {
+    "arrkom": "pk_test_replace_this",
+    "prokom": "pk_test_replace_this",
+    "trikom": "pk_test_replace_this"
+}
+
+STRIPE_PRIVATE_KEYS = {
+    "arrkom": "sk_test_replace_this",
+    "prokom": "pk_test_replace_this",
+    "trikom": "pk_test_replace_this"
```

Comment:
One is prefixed sk and the others are prefixed pk? Be consistent please, so people who want to debug this uses the correct key.